src/sentry/web/frontend/auth_login.py

```diff
@@ -504,22 +505,29 @@ def can_register(self, request: Request) -> bool:
     def get_default_context(self, request: Request, **kwargs) -> dict:
         """
         Sets up a default context that will be injected into our login template.
+        TODO: clean up unused context
         """
         organization = kwargs.pop("organization", None)
         default_context = {
             "server_hostname": get_server_hostname(),
             "login_form": None,
-            "organization": kwargs.pop("organization", None),
+            "organization": kwargs.pop(
+                "organization", None
+            ),  # NOTE: not utilized in basic login page (only org login)
```

Comment:
The org login _page_ is the same as the basic login page isn't it? Would it be more accurate to say something like `Only used for org login`?

Reply:
Not quite - the org login view subclasses this login view, but it's not the same login view